exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/QueryResources.java

```diff
@@ -128,9 +133,9 @@ public int getDefaultRowsAutoLimited() {
     private final List<String> columns;
     private final List<List<String>> rows;
     private final String queryId;
-    private final int defaultRowsPerPage;
+    private final String defaultRowsPerPage;
```

Comment:
Same here and below. Rename`defaultRowsPerPage` --> `rowsPerPageOptions`

Reply:
+1

Reply:
Looks like you missed updating the name here.

Reply:
Yes.. missed it. Fixed it all around.